digikeyscraper/core.py

```diff
@@ -233,16 +238,17 @@ def remove_ocr(pdf_dir):
     """
     Use `pdffonts` to remove PDFs that are just images and would require OCR.
     """
-    logger.info("Begin removing PDFs that need OCR...")
+    logger.info("Removing PDFs that need OCR...")
     count = 0
     src_files = os.listdir(pdf_dir)
-    for pdf in src_files:
-        pdf_filename = os.path.join(pdf_dir, pdf)
-        if check_ocr(pdf_filename):
-            logger.debug("OCR Filtering: " + pdf_filename)
-            os.remove(pdf_filename)
-            count += 1
-
+    with tqdm(total=len(src_files)) as pbar:
+        for pdf in src_files:
+            pdf_filename = os.path.join(pdf_dir, pdf)
+            if check_ocr(pdf_filename):
+                logger.debug("OCR Filtering: " + pdf_filename)
```

Comment:
Also suggest using f-strings
```suggestion
                logger.debug(f"OCR Filtering: {pdf_filename}")
```